.github/workflows/ci.yaml

```diff
@@ -56,12 +46,39 @@ jobs:
           push: true
           file: ./Dockerfile
           build-args: VERSION=${{ env.VERSION }}
-          cache-from: ${{ env.IMAGE_REPOSITORY }}:buildcache
           tags: |
-            ${{ env.IMAGE_REPOSITORY }}:${{ env.VERSION}}
-            ${{ env.IMAGE_REPOSITORY }}:buildcache
+            ${{ secrets.ASTRO_IMAGE_REPOSITORY }}:${{ env.VERSION }}
+            ${{ secrets.EC_IMAGE_REPOSITORY }}:${{ env.VERSION }}
 
-      - name: Trigger update to staging Fargate cluster
+      -
+        name: Configure EarthChem AWS credentials
+        id: ec-creds
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.DEPLOYER_EC_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.DEPLOYER_EC_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+           
+      -
+        name: Trigger update to EarthChem staging Fargate cluster
+        run: |
+          aws cloudformation update-stack \
+            --stack-name staging-pid-service-fargate-service \
+            --parameters ParameterKey=ImageTag,ParameterValue=${{ env.VERSION}} \
+            ParameterKey=ServiceName,UsePreviousValue=true \
+            ParameterKey=Environment,UsePreviousValue=true \
+            --use-previous-template || true
+      -
+        name: Configure Astromat AWS credentials
+        id: astro-creds
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.DEPLOYER_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.DEPLOYER_AWS_SECRET_ACCESS_KEY }}
```

Comment:
Shouldn't this be called `DEPLOYER_ASTRO_AWS...` to be consistent with EC?

Reply:
Resolved by https://github.com/amds-ldeo/pid-service/pull/5/commits/1ce69e0fbfd9e068c4afe5a5c84bdf9ffd7e39c9